### ISSUE TITLE
Make more sense of distances for multi-centered searches

### ISSUE THREAD

**Issue by GoogleCodeExporter** [steve8x8/geotoad#316]
```
When running a multi-centered search (-q coord a,b:c,d:e,f), the 
relativedistance{,km} values seem to be random. They are not, in a strict 
sense, as they are parsed from the result page(s), but multiple center points 
result in different distances, of course.

In a multi-center search distances should be 
(a) invalidated completely ("N/A")
or
(b) re-calculated based on a single origin (how to specify that? the last one 
used?)
or
(c) calculate all distances relative to "my home" (how to access that?)
or ...

I'm leaving this "bug" report as a reminder. Feel free to comment.
```

Original issue reported on code.google.com by `Steve8x8` on 10 Mar 2015 at 3:22

**Comment by steve8x8** [comment]
What about this: 
- use the distances shown on the search page (which may change from run to run - apparently different servers use different earth models and/or rounding techniques), as before 
- unless there's a set of (purely numeric) GEO_HOME_LAT and GEO_HOME_LON environment variables, in which case distances (and possibly azimuths!) will be re-computed (formula still to be determined).
If those env variables aren't set, the behaviour won't change; no need for yet another command line option (and parsing/conversion); any drawbacks I'm overlooking?

**Comment by steve8x8** [comment]
This could be enhanced: 
  * If the env variables aren't set, nothing will change wrt the old behaviour.
  * If both variables are explicitly set to zero (very wet place), the user's home location is queried from GC.
  * if the env variables are set and not both are zero, they override all distance calculations.

Comments?

**Comment by steve8x8** [comment]
This has been committed a while ago, and will be in 3.23.2

**Comment by steve8x8** [comment]
Released with 3.24.0.

Leaving this issue open (although considered resolved) for comments and suggestions.

**Comment by steve8x8** [comment]
After 3.25.0 release, closing.
